docs/content/guides/operations/kubernetes/kubernetes-install/index.md

```diff
@@ -23,6 +23,14 @@ rad install kubernetes
 rad install kubernetes --set global.zipkin.url=http://jaeger-collector.radius-monitoring.svc.cluster.local:9411/api/v2/spans,rp.publicEndpointOverride=localhost:8081
 ```
 
+### Use your own root certificate authority certificate
+
+Many enterprise users leverage intermediate root certificate authorities (CAs) to enhance security and control over outgoing traffic originating from their employees' machines, particularly when using a firewall solution. In such environment, when Radius attempts to connect to external endpoints, such as Azure, AWS control planes, or IAMs, all traffic are blocked by the firewall. You can optionally use the `--set-file` to inject the root CA certificates to Radius containers:
```

Comment:
```suggestion
Many enterprises leverage intermediate root certificate authorities (CAs) to enhance security and control over outgoing traffic originating from their employees' machines, particularly when using a firewall solution. In this setup, when Radius attempts to connect to an external endpoint, such as Azure or AWS, traffic is blocked by the firewall. You may optionally use`--set-file` when installing Radius to inject your root CA certificates into Radius:
```

Reply:
Do we need to a position on why enterprises do this? I would argue that it doesn't enhance security for the user.

Reply:
We need to mention proxies in here, and we should probably keyword/tag the term proxy so it can be searched. 

This is a proxy not a firewall 😆

Reply:
> Do we need to a position on why enterprises do this? I would argue that it doesn't enhance security for the user.

The intermediate root CA is being used for the security in the corp network. 

> We need to mention proxies in here, and we should probably keyword/tag the term proxy so it can be searched.
> 
> This is a proxy not a firewall 😆

Root CA is controlled by firewall.

Reply:
@rynowak - please take a look at the updated text that incorporates your feedback and @youngbupark's answers. if this looks good to you, please provide your approval or else let me know if additional feedback: https://github.com/radius-project/docs/pull/958/files#r1406909453